service-runner/src/main/resources/config/assets.yaml

```diff
@@ -58,6 +58,61 @@ assets:
     sep24_enabled: true
     sep31_enabled: true
     sep38_enabled: true
+  - schema: stellar
```

Comment:
There are two `stellar:USDC` defined in the `assets.yaml`. Can we merge them?

Reply:
Fireblocks supports USDC with another issuer, that's why we added new USDC:
![image](https://github.com/stellar/java-stellar-anchor-sdk/assets/131388170/e0c7e18b-df72-4a30-9d8a-a33d7b20a609)